vendor/http_parser/http_parser.c

```diff
@@ -359,7 +359,7 @@ enum state
   };
 
 
-#define PARSING_HEADER(state) (state <= s_headers_done)
+#define PARSING_HEADER(state) ((state) <= s_headers_done)
```

Comment:
This is a vendored library and shouldn't be changed. Any changes should be submitted to the upstream project.

Reply:
@SadieCat, fix it.
Commit info here:
https://github.com/inspircd/inspircd/pull/1879/commits/b4e0c5b6212def02386af0a555445412eed6f1f4